src/ui/qgsoptionsbase.ui

```diff
@@ -4356,6 +4356,13 @@ The bigger the number, the faster zooming with the mouse wheel will be.</string>
                     </property>
                    </widget>
                   </item>
+                  <item row="4" column="0">
+                   <widget class="QLabel" name="mDefaultMValueLabel">
+                    <property name="text">
+                     <string>Default M value</string>
```

Comment:
"Default M(easure) value" ? Or add a tooltip to explain what M means for non-SF literate people :-)

Reply:
Right! I've just added a tool tip for both spinboxes
![image](https://user-images.githubusercontent.com/7521540/116664249-7ea81580-a998-11eb-9224-a292896d4c7e.png)